### PR TITLE
adding types selects to email, impp and tel

### PR DESCRIPTION
**Pull request by DeepDiver1975**
refs #99

### CODE REVIEW

File: js/public/script.js [hunk 1]
```diff
@@ -1154,17 +1154,39 @@ app.service('vCardPropertiesService', [function() {
...
+			template: 'text',
+			defaultValue: {
+				value:[''],
+				meta:{type:['HOME']}
+			},
+			options: [
+				{id: 'HOME', name: t('contacts', 'Home')},
+				{id: 'WORK', name: t('contacts', 'Work')},
+				{id: 'OTHER', name: t('contacts', 'Other')}
+			]
```
Comment (irgendwie) [inline js/public/script.js:1180]: Instant messaging should be separated by the protocol (AIM, Jabber, MSN, etc.) and not by categories (Home, Work, etc.)!

Comment (DeepDiver1975) [inline js/public/script.js:1180]: the protocol is part of the value - https://tools.ietf.org/html/rfc6350#section-6.4.3

Comment (irgendwie) [inline js/public/script.js:1180]: whoops, my fault - sry!

**Comment by irgendwie** [comment]
LGTM :+1:

**Pull request merged**